Memory/src/main/scala/com/thoughtworks/compute/Memory.scala

```diff
@@ -36,6 +36,8 @@ trait Memory[Element] {
 
 object Memory extends LowPriorityMemory {
 
+  final val CHAR_ALLOC_DEFAULT_BYTE_SIZE = 2
```

Comment:
You should use `numberOfBytesPerElement`, instead of defining this `CHAR_ALLOC_DEFAULT_BYTE_SIZE`